src/utils/path/interpolators/path-points.js

```diff
@@ -369,7 +369,7 @@ function interpolatePoint(a, b, t) {
         return b;
     }
     var c = {};
-    var props = utils.unique(Object.keys(a), Object.keys(b));
+    var props = Object.keys(a);
```

Comment:
That’s a weird way to contact you, but I have no other way to find you bro. I need to contact you desperately. ac_pepe in telegram